lib/Base/Organization/CollectionCalls.vala

```diff
@@ -0,0 +1,81 @@
+/* CollectionCalls.vala
+ *
+ * Copyright 2023 Frederick Schenk
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+using GLib;
+
+/**
+ * An interface for Collection providing access methods for pullable lists.
+ */
+public interface Backend.PullableCollection<T> : Backend.Collection<T> {
```

Comment:
I'll have to think about how these interfaces work alongside the streaming feed.

Reply:
Since a `ListView` would be notified automatically about changes to a `ListModel`, I don't think we need an additional method for streamed collections, only an property with which the frontend can decide if it want's to stream the data. Said property should probably be in an additional interface.